cmd/serve/serve.go

```diff
@@ -128,6 +131,15 @@ func serveCmd(cmd *cobra.Command, args []string) error {
 			excludes = append(excludes, re)
 		}
 	}
+	// parse proxy url
+	var proxyUrl *url.URL = nil
```

Comment:
```suggestion
	var proxyUrl *url.URL
```
Pointers are nil by default.